client/src/components/Login.js

```diff
@@ -22,9 +22,9 @@ export class Login extends Component {
   handleSubmit = event => {
     event.preventDefault();
     const { email, password } = this.state;
-
     getToken(email, password)
       .then(res => {
+        sessionStorage.languages = JSON.stringify(res.languages);
```

Comment:
It works but maybe you can use `setItem` too, just for consistency.

Reply:
done